### DISCUSSION TITLE
terminate called after throwing an instance of 'std::out_of_range'

### DISCUSSION THREAD

**Discussion by wh44496** [hahnlab/CAFE5#171] [Q&A] [answered]
how could I solve this problem?
"terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 720) >= this->size() (which is 720)
Aborted"
thank you for your help!

**Comment by benfulton** [comment, marked as answer]
Probably making your families smaller would fix it. There are some hints [here](https://github.com/hahnlab/CAFE5/issues/65#issuecomment-952419357).
